packages/ra-ui-materialui/src/layout/ApplicationUpdatedNotification.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Alert, AlertProps, Button, ButtonProps } from '@mui/material';
+import { useTranslate } from 'ra-core';
+
+export const ApplicationUpdatedNotification = React.forwardRef<
+    HTMLDivElement,
+    ApplicationUpdatedNotificationProps
+>((props, ref) => {
+    const {
+        ButtonProps,
+        updateText = 'ra.action.update_application',
+        notificationText = 'ra.notification.application_update_available',
+        ...alertProps
+    } = props;
+    const translate = useTranslate();
+
+    const handleButtonClick = () => {
+        window.location.reload();
+    };
+    return (
+        <Alert
+            ref={ref}
+            severity="info"
+            action={
+                <Button
+                    color="inherit"
+                    size="small"
+                    onClick={handleButtonClick}
+                    {...ButtonProps}
+                >
+                    {translate(updateText, { _: updateText })}
```

Comment:
What is the benefit of providing a default translation that is equal to the translation key? :thinking:

Reply:
It comes from a prop. Users may pass a simple string that is not a translation key